af/analysis/rotation.py

```diff
@@ -24,6 +30,7 @@
     `ipaddr` IPv4,
     `ipv6addr` IPv6,
     `draining_at` Nullable(DateTime('UTC')),
+    `destroyed_at` Nullable(DateTime('UTC')),
     `sign` Int8
 )
 ENGINE = CollapsingMergeTree(sign)
```

Comment:
If we are effectively always using sign = 1, why don't we at that point just use a ReplacingMergeTree?